build/commands/lib/config.js

```diff
@@ -361,7 +361,7 @@ Config.prototype.buildArgs = function () {
     args.last_chrome_installer = this.last_chrome_installer
   }
 
-  if (process.platform === 'darwin') {
+  if (process.platform === 'darwin' || this.targetOS === 'mac' || this.targetOS === 'ios') {
```

Comment:
I don't think `ios` can be built that way at all, so we shouldn't support it here.

Reply:
Fixed in d610a52518070148ce5e65708fe065f65d4df3b8.